src/main/java/ch/bfh/bti7081/view/SeminarView.java

```diff
@@ -52,19 +56,19 @@
     private String googleApiKey;
 
     private H1 title = new H1("Seminarfinder");
-    private VerticalLayout welcomeLayout = new VerticalLayout(title);
 
-    private ComboBox<SeminarCategory> categoriesCb = new ComboBox<>("Kategorie:");
+    private ComboBox<String> categoriesCb = new ComboBox<>("Kategorie:");
```

Comment:
Ich habe bei mir schon letztes mal festgestellt, dass die Elemente in der Kategorie-Auswahl länger haben um zu erscheinen. 
Zumindest bei mir habe ich das nachgeprüft und die ComboBox mit einem Select ersetzt. Dann sind alle Kategorien schon beim Klick da. 
Wäre:
private Select<String> categoriesCb = new Select<>();
Und unter den ElementSetting noch folgendes:
categoriesCb.setLabel("Kategorie");

Reply:
Hani ibout. Funktioniert itse viu besser! danke!